content/rust/docker/rust-docker-arm.md

```diff
@@ -12,3 +12,365 @@ tags:
 - docker
 - arm
 ---
+
+This guide will walk you through the steps required to build and publish arm64 Docker images for Rust projects. This guide assumes you have a basic understanding of Rust and Docker, and that you have Rust and Docker installed.
+
+# Setup a Rust Project
+
+First we setup an example project. To do this we can use `cargo`:
+
+```
+$ cargo init hello-world
+```
+
+This will create a folder named `hello-world` which contains the project. It consists of a project file `Cargo.toml` and a source file `main.rs` in the `src` subdirectory. The source file contains code which simply prints `Hello, world!` in the console:
```

Comment:
`s/will create/creates/g`

In general, omit the word `will` and use the present tense not the future.